### PR TITLE
Proxy support

### PR DESCRIPTION
**Pull request by Adizbek**
Usage

```js

const t = require('@k3rn31p4nic/google-translate-api');

const resp = t('Hello', {
  from: 'en',
  to: 'ru',

  proxies: [
    { host: '134.209.1.204', port: 8080 }
  ]
});

resp.then(x => {
  console.log(x);
}).catch(x => {
  console.log(x);
});

```

I didn't check proxy above

### CODE REVIEW

**Comment by Adizbek** [comment]
I thought, picking random proxy is better because it will decrease chance getting blocked.

**Comment by k3rn31p4nic** [comment]
Nevermind, I didn't notice your 2nd commit and that you were already exposing the `proxies` option to the user.

**Comment by ZuBB** [comment]
I have tried to test this code for my case with help of https://github.com/chill117/proxy-lists and https://github.com/chill117/proxy-verifier but failed to succeed. My guess is that I did not pick right proxy

> I didn't check proxy above

Hope you had a successful run of this PR

File: src/index.js [hunk 1]
```diff
@@ -78,6 +79,16 @@ async function translate(text, options) {
       requestOptions = [ url ];
     }
 
+    let proxies = [];
```
Comment (ZuBB) [inline src/index.js]: can be moved inside of `if`

File: src/index.js [hunk 2]
```diff
@@ -78,6 +79,16 @@ async function translate(text, options) {
...
+    if (Array.isArray(options.proxies)) {
+      proxies = options.proxies;
+
+      requestOptions[1] = {
+        agent: tunnel.httpOverHttp({
+          proxy: proxies[Math.floor(Math.random() * proxies.length)]
```
Comment (ZuBB) [inline src/index.js]: not sure how this whole thing will work in case of empty array will be passed..

**Comment by Adizbek** [comment]
@ZuBB I think found the solution. Proxy didn't give any benefit. #10 just look at this PR, I am using this on my server for now, works great. 

Try 
```
npm install github:Adizbek/google-translate-api#32cb90c49792c3a65c6893e2c3487b1a3a01c1e0 --save
```

**Comment by MaxySpark** [comment]
@Adizbek so, Is there any way to use proxy now

**Comment by Adizbek** [comment]
No, as I am not facing this problem anymore, I gave up working on it.

**Pull request merged**